tasks/config/run.js

```diff
@@ -6,6 +6,8 @@ module.exports = function (grunt) {
   const root = p => resolve(__dirname, '../../', p);
   const binScript =  /^win/.test(platform) ? '.\\bin\\kibana.bat' : './bin/kibana';
   const buildScript =  /^win/.test(platform) ? '.\\build\\kibana\\bin\\kibana.bat' : './build/kibana/bin/kibana';
+  const pkgVersion = grunt.config.get('pkg.version');
+  const releaseBinScript = `./build/kibana-${pkgVersion}-linux-x86_64/bin/kibana`;
```

Comment:
can we make this dynamic?  a combination of os.platform() and os.arch() should get us there